### PR TITLE
[docker] Add quilt configuration files to container

### PR DESCRIPTION
**Pull request by tsirakisn**
Two files added to build user $HOME:
* .quiltrc
* oxt-patch.header

Apply patch header by running the following within
a quilt-series directory (applies to current patch):

`quilt header -r < ~/oxt-patch.header`

Note that -r replaces any header that existed before it.
You can use -e to edit the header in your $EDITOR.

Signed-off-by: Nicholas Tsirakis <tsirakisn@ainfosec.com>

### CODE REVIEW

File: Dockerfiles/openxt-buster-oe64 [hunk 1]
```diff
@@ -50,6 +50,10 @@ RUN ln -s /lib64/ld-linux-x86-64.so.2 /lib/
 
 RUN useradd -Ums /bin/bash -l -p '""' -u $UID $UNAME
 
+# Add quilt config files
+ADD root/home/build/.quiltrc /home/build/.quiltrc
+ADD root/home/build/openxt/oxt-patch.header /home/build/openxt/oxt-patch.header
```
Comment (jandryuk) [inline Dockerfiles/openxt-buster-oe64]: Docker mounts $TOP, which is the openxt manifest directory, over /home/build/openxt: https://github.com/OpenXT/bordel/blob/d9c52e1bab6c7e5a36241df24838e66ce79579f4/cmds/docker#L60 Does oxt-patch.header still show inside the container?

Comment (tsirakisn) [reply Dockerfiles/openxt-buster-oe64]: Ah you're right, my testing was flawed. Misused the docker command. I'm just going to put it in the home dir in that case.

Comment (jandryuk) [reply Dockerfiles/openxt-buster-oe64]: Ok, sounds good.

**Review by jandryuk — APPROVED** [review]

**Review by crogers1 — APPROVED** [review]
LGTM, merging soon

**Comment by crogers1** [comment]
> LGTM, merging soon

That was a liberal use of "soon".  Merging.

**Pull request merged**